distributed/protocol/pickle.py

```diff
@@ -1,17 +1,8 @@
 import logging
-import sys
+import pickle
 
 import cloudpickle
 
-if sys.version_info < (3, 8):
-    try:
-        import pickle5 as pickle
-    except ImportError:
-        import pickle
-else:
-    import pickle
```

Comment:
Now that pickle protocol 5 is builtin, it should be possible to drop all of the `protocol=4` lines like those that were added here ( https://github.com/dask/distributed/pull/4019 )

Reply:
Sorry @jakirkham, I missed this comment when you originally left it. I agree we probably don't need several of those `protocol=4` anymore. That said, due to my bandwidth limitations I'd prefer to have those addressed in a separate PR. It would be good to have them cleaned up, but things will also work as-is on Python 3.8+

Reply:
Happy to pick up as a separate PR. Looking forward to pickle protocol 5 as the default 🙂

Reply:
Added draft PR ( https://github.com/dask/distributed/pull/5826 ) with this change